python/sdk/prelude_sdk/controllers/iam_controller.py

```diff
@@ -47,13 +49,15 @@ def purge_account(self):
         raise Exception(res.text)
 
     @verify_credentials
-    def update_account(self, mode: Mode = None, company: str = None):
+    def update_account(self, mode: Mode = None, company: str = None, slug: str = None):
         """ Update properties on an account """
         body = dict()
         if mode is not None:
             body['mode'] = mode.name
         if company is not None:
             body['company'] = company
+        if slug is not None:
```

Comment:
why isn't this just "if slug" ? To match the previous style?

Reply:
There are two things that a user could want to do:
1) update other fields and leave slug as is
2) set slug to None

In the CLI:
The first is done by not using the `--slug` flag at all, which defaults the `slug` value to None.
To differentiate from that, the second is going to be done by setting the `--slug` flag to `''` 

So in the SDK:
If `slug` is None, we don't set the `slug` field at all in our API request, which means don't touch that field in the BE
If `slug` is empty string, the `slug` field will be in our API request, and the empty string will convert to `None` in the BE

It isn't the cleanest solution, but it works. If you can think of a better way to differentiate, I'm open!